### PR TITLE
Precaution table

### PR DESCRIPTION
**Pull request by DimaDiD**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/api/distinctionApi.tsx [hunk 1]
```diff
@@ -16,13 +17,26 @@ const getUserDistinctions = async () => {
   return (await api.get(`Distinction/UserDistinctions`)).data;
 };
 
-const getAllUsersDistinctions = async (searchedData: string, page: number, pageSize: number) => {
+/*const getAllUsersDistinctions = async (searchedData: string, page: number, pageSize: number) => {
```
Comment (VSBalanchuk) [inline src/api/distinctionApi.tsx:20]: It is best practise to remove commented code. Cause it can cause a lot of confusion later. 
So, please, remove it

**Review by NatalyMinziuk — APPROVED** [review]

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 2]
```diff
@@ -10,7 +10,8 @@ import EditDistinctionTypesModal from "./EditDistinctionTypesModal";
 import UserDistinctionTableInfo from "../Interfaces/UserDistinctionTableInfo";
 import ClickAwayListener from "react-click-away-listener";
 import Distinction from "../Interfaces/Distinction";
-import Spinner from "../../Spinner/Spinner";
+import DistionctionTableSettings from "../../../models/Distinction/DistinctionTableSettings";
+//import Spinner from "../../Spinner/Spinner";
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:14]: Same here

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 3]
```diff
@@ -46,7 +47,8 @@ const DistinctionTable = () => {
   const [page, setPage] = useState(1);
   const [pageSize, setPageSize] = useState(10);
   const [total, setTotal] = useState<number>(0);
-  const [count, setCount] = useState<number>(0);
+  //const [count, setCount] = useState<number>(0);
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:50]: And here

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 4]
```diff
@@ -71,19 +73,26 @@ const DistinctionTable = () => {
...
+      searchedData: searchedData,
+      page: page,
+      pageSize: pageSize
+    };
+
+    setLoading(true);
+    const res: UserDistinctionTableInfo[] = await distinctionApi.getAllUsersDistinctions(NewTableSettings);    
     setTotal(res[0]?.total);
-    setCount(res[0]?.count);
+    //setCount(res[0]?.count);
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:87]: And here

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 5]
```diff
@@ -100,13 +109,12 @@ const DistinctionTable = () => {
 
   const handleAdd = async () => {
     setVisibleModal(false);
-    setLoading(true);
-    const res: UserDistinctionTableInfo[] = await distinctionApi.getAllUsersDistinctions(searchedData, page, pageSize);
+    /*const res: UserDistinctionTableInfo[] = await distinctionApi.getAllUsersDistinctions(searchedData, page, pageSize);
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:112]: And here

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 6]
```diff
@@ -117,14 +125,14 @@ const DistinctionTable = () => {
     setShowDropdown(false);
   };
 
-  const handlePageChange = (page: number) => {
+  /*const handlePageChange = (page: number) => {
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:128]: And here

File: src/pages/Distinction/DistinctionTable/DistinctionTable.tsx [hunk 7]
```diff
@@ -270,29 +289,31 @@ const DistinctionTable = () => {
...
                 current: page,
                 pageSize: pageSize,
-                total: count,
+                total: total,
                 showLessItems: true,
                 responsive: true,
                 showSizeChanger: true,
-                onChange: (page) => handlePageChange(page),
-                onShowSizeChange: (page, size) => handleSizeChange(page, size),
+                //onChange: (page) => handlePageChange(page),
```
Comment (VSBalanchuk) [inline src/pages/Distinction/DistinctionTable/DistinctionTable.tsx:308]: And here

File: src/api/distinctionApi.tsx (hunk 1 above)
Comment (DimaDiD) [reply src/api/distinctionApi.tsx]: Done

**Review by VSBalanchuk — APPROVED** [review]

**Pull request closed without merging**